src/compiler/commandLineParser.ts

```diff
@@ -446,6 +446,9 @@ namespace ts {
                             }
                             if (opt.isFilePath) {
                                 value = normalizePath(combinePaths(basePath, value));
+                                if (value === "") {

```

Comment:
Then probably we have to use `typeof options.somePathOption === 'string'` everywhere when we need to check is a path option is specified. Currently zero length path options are treated as not specified...